packages/core-api/src/app/App.tsx

```diff
@@ -222,61 +223,11 @@ export class PrivateAppImpl implements BackstageApp {
   }
 
   getRoutes(): JSX.Element[] {
-    const routes = new Array<JSX.Element>();
-
-    const { NotFoundErrorPage } = this.components;
-
-    for (const plugin of this.plugins.values()) {
-      for (const output of plugin.output()) {
-        switch (output.type) {
-          case 'legacy-route': {
-            const { path, component: Component } = output;
-            routes.push(
-              <Route key={path} path={path} element={<Component />} />,
-            );
-            break;
-          }
-          case 'route': {
-            const { target, component: Component } = output;
-            routes.push(
-              <Route
-                key={`${plugin.getId()}-${target.path}`}
-                path={target.path}
-                element={<Component />}
-              />,
-            );
-            break;
-          }
-          case 'legacy-redirect-route': {
-            const { path, target } = output;
-            routes.push(<Navigate key={path} to={target} />);
-            break;
-          }
-          case 'redirect-route': {
-            const { from, to } = output;
-            routes.push(<Navigate key={from.path} to={to.path} />);
-            break;
-          }
-          default:
-            break;
-        }
-      }
-    }
-
-    routes.push(
```

Comment:
This one would need to be added to `FlatRoutes`, I guess? 🤔

Reply:
Already there :grin:

This is actually the wrong version though, didn't intend to remove this block xP. This can stay and will be backwards compatible, just that routes won't be discovered from plugins that are discovered through the react tree, but what's the point of that anyway? :grin: